td.vue/src/components/GraphButtons.vue

```diff
@@ -123,6 +129,9 @@ export default {
                 this.graph.showGrid();
                 this.gridShowing = true;
             }
+        },
+        goToHome() {
+            this.$router.push('/'); // Navigate to the home route
```

Comment:
we would need to check for modification of the diagram before exiting to the landing page